src/utils/projectTypes.ts

```diff
@@ -0,0 +1,31 @@
+import "@formatjs/intl-listformat/locale-data/en";
+import "@formatjs/intl-listformat/polyfill";
+
+import { getValidSchemaDictionary } from "./digitalPlanningSchema";
+
+export function formatRawProjectTypes(rawProjectTypes: string[]): string {
+  const schemaProjectTypes = getValidSchemaDictionary("ProjectType");
+  if (schemaProjectTypes) {
+    const matchingDescriptions = Object.entries(schemaProjectTypes)
+      .filter(([key, _description]) => rawProjectTypes.includes(key))
+      .map(([_key, description]) => description.toLocaleLowerCase());
+    if (matchingDescriptions.length === 0) {
+      return "Unknown";
+    }
+    return formatHumanReadableProjectTypes(matchingDescriptions);
+  } else {
+    return "Unknown";
```

Comment:
Projects throughout email templates will now fall into one of three cases: 
1) "Project type not submitted" if the passport does not contain `proposal.projectType`
2) List of human-readable project types based on schema if passport does contain `proposal.projectType`
3) "Unknown" if passport does contain `proposal.projectType` but none of those values are valid schema values

I think we should leave "Unknown" as-is for this first refactor and see if/how it shifts feedback. If we're still receiving questions or concerns, there's a few ideas for what we could differently:
- Only return raw project types that are a single segment (eg `breach` &rarr; `Breach`)
- Return all raw project types instead of "Unknown" (either as dot-separated raw variables or perhaps replacing `.` with space and applying formatting?)
- Improve Airbrake logging for these cases (in most cases they are going to be values we _do not_ want to add to schema though, so hesitant to introduce this noise now)

Reply:
This is a solid plan 👍